src/fs_event_wrap.cc

```diff
@@ -104,7 +102,7 @@ void FSEventWrap::Initialize(Local<Object> target,
 
   AsyncWrap::AddWrapMethods(env, t);
   env->SetProtoMethod(t, "start", Start);
-  env->SetProtoMethod(t, "close", Close);
+  env->SetProtoMethod(t, "close", HandleWrap::Close);
```

Comment:
I'd use unqualified `Close`, that turns it into a build error if `FSEventWrap` stops being a `HandleWrap`.